test/org/traccar/protocol/Jt600ProtocolDecoderTest.java

```diff
@@ -18,13 +18,13 @@ public void testDecode() throws Exception {
                 "2475605035891613002328091601152806086750106533350c00000000000a000000000000e1ff4f97007f1607"));
 
         verifyPosition(decoder, buffer(
-                "(3301210003,U01,040812,185302,T,22.564025,N,113.242329,E,5.21,152,9,32%,00000000000011,10133,5173,22,100,1)"));
+                "(3301210003,U01,040812,185302,T,22.564025,N,113.242329,E,5.21,152,9,32%,00000000000011,10133,5173,22,100,1,18)"));
```

Comment:
Please don't modify existing test cases because they are taken from real devices or documentation, so they are valid messages. Add new ones if required.